src/components/AddNote/AddNote.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../scss/variables.scss";
+@import "../card/Card.scss";
+
+.add-note-container{
```

Comment:
We'll also want to be able to have a scrollbar for the notes container, so users can scroll down to view all of them

Reply:
Just want to clarify.  Do you mean to add a scroll bar so the existing notes container so the user can view all of the notes while adding a note?  I think that would probably be a good thing to add, since currently the ticket modal doesn't appear to have any handling for that.  

I played around with some changes to the Ticket Modal and Ticket Modal.scss that I think achieves this goal.  Sorry about all of the little commits.

Reply:
It looks great when there are many notes on a Ticket, but when there is only a couple notes, we might want to adjust the spacing and remove the scrollbar until it is needed
![image](https://user-images.githubusercontent.com/8600792/118012824-4ee8fe00-b306-11eb-84db-1695147b80ab.png)

Reply:
Also do you mind adding parentheses around the number of notes? I think it's clearer to have Notes (5) vs Notes 5